From a6b26535bb0b94309377479ccd4ad43d074ef9ae Mon Sep 17 00:00:00 2001
From: David Seifert <soap@gentoo.org>
Date: Fri, 13 Jan 2017 16:12:42 +0100
Subject: [PATCH 3/4] Honor user's $(htmldir) and do not override GNU defaults
 for $(docdir)

* HTML files should be installed to $(htmldir), and $(docdir) should
  not be changed, as this is a user flag in the GNU conventions.
---
 doc/Makefile.am             |  2 --
 doc/html/Makefile.am        | 10 ++++------
 doc/html/images/Makefile.am |  4 ++--
 3 files changed, 6 insertions(+), 10 deletions(-)

diff --git a/doc/Makefile.am b/doc/Makefile.am
index fb79e3a7..bc9ae52a 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -31,8 +31,6 @@ FLAC.tag:
 	mkdir -p html/api
 endif
 
-docdir = $(datadir)/doc/$(PACKAGE)-$(VERSION)
-
 doc_DATA = \
 	FLAC.tag
 
diff --git a/doc/html/Makefile.am b/doc/html/Makefile.am
index 813b55f2..ef4a12d8 100644
--- a/doc/html/Makefile.am
+++ b/doc/html/Makefile.am
@@ -18,9 +18,7 @@
 
 SUBDIRS = images
 
-docdir = $(datadir)/doc/$(PACKAGE)-$(VERSION)/html
-
-doc_DATA = \
+html_DATA = \
 	changelog.html \
 	developers.html \
 	documentation.html \
@@ -46,10 +44,10 @@ if FLaC__HAS_DOXYGEN
 # The install targets don't copy whole directories so we have to
 # handle 'api/' specially:
 install-data-local:
-	$(mkinstalldirs) $(DESTDIR)$(docdir)/api
-	(cd $(builddir)/api && $(INSTALL_DATA) * $(DESTDIR)$(docdir)/api)
+	$(mkinstalldirs) $(DESTDIR)$(htmldir)/api
+	(cd $(builddir)/api && $(INSTALL_DATA) * $(DESTDIR)$(htmldir)/api)
 uninstall-local:
-	rm -rf $(DESTDIR)$(docdir)/api
+	rm -rf $(DESTDIR)$(htmldir)/api
 distclean-local:
 	-rm -rf api
 endif
diff --git a/doc/html/images/Makefile.am b/doc/html/images/Makefile.am
index 33485f3a..f8f551da 100644
--- a/doc/html/images/Makefile.am
+++ b/doc/html/images/Makefile.am
@@ -16,9 +16,9 @@
 #  restrictive of those mentioned above.  See the file COPYING.Xiph in this
 #  distribution.
 
-docdir = $(datadir)/doc/$(PACKAGE)-$(VERSION)/html/images
+logosdir = $(htmldir)/images
 
-doc_DATA = \
+logos_DATA = \
 	logo.svg \
 	logo130.gif
 
-- 
2.11.0

